packages/nimble-components/src/select/specs/IxD.md

```diff
@@ -0,0 +1,254 @@
+
+Keep the label text as concise as possible, limited to a single line of text. Use sentence casing.
+
+In a form, use the label to indicate whether the select is optional or required for the end-user to interact with. If most of the components in the form are required, indicate which components are optional. If most of the components in the form are optional, indicate which components are required. Required or optional indication in the label text should always follow the pattern "[Label] (optional)" or "[Label] (required)".
+
+![ ](spec-images/select-2.png)
+
+#### Helper Text
+> NOTE: NEEDS VISUAL DESIGN FOR TYPOGRAPHY
+
+Only include helper text to provide information that aids the end-user in choosing the right selection. Keep the helper text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-3.png)
+
+#### Field Button
+Selects can be empty by default or have a default selection from the menu. Empty selects should include placeholder text that propts the end-user to select an option from the menu. Placeholder text should always follow the pattern "Select [thing(s)]", for example "Select country". Ellipses are not needed. Use sentence casing.
+
+![ ](spec-images/select-4.png)
+
+#### Menu & Menu Items
+A list of options to select, displayed as an open state. Order menu items alphabetically by default, unless another ordering will provide value to the end-user. Keep the menu item text concise as possible, limited to a single line of text. Use sentence casing.
+
+![ ](spec-images/select-5.png)
+
+### Related Components
+To allow end-users to enter a custom or create a new menu item, use a [**combobox**](/packages/nimble-components/src/combobox/specs/IxD.md).
+
+![ ](spec-images/select-alternative-combobox.png)
+
+> NOTE: SEE FUTURE CONSIDERATIONS FOR MULTI-SELECTION
+
+To allow end-users to multi-select menu items, use a [**tag picker**](/packages/nimble-components/src/tag-picker/specs/IxD.md).
+
+![ ](spec-images/select-alternative-picker.png)
+
+## Design
+### Configuration
+#### Label
+> NOTE: NEEDS VISUAL DESIGN FOR PLACEMENT
+
+The client-user should be able to specify whether they want the label to be positioned at the top or to the left of the select or not at all. Selects that don't have a label should still have an ARIA label.
+
+![ ](spec-images/select-6.png)
+
+#### Icons
+The client-user should be able to specify whether they want left icons on menu items or not. Icons show up in menu items and are displayed as part of the value in the field button. Images and other content not specified here should not be permitted.
+
+![ ](spec-images/select-7.png)
+
+#### Groups
+> QUESTION: IS THE SL "VIEW" PICKER A SELECT (SHOULD IT BE)? WHAT ABOUT GROUPS WITHOUT LABELS, WHAT ABOUT A SELECTION GROUP VS. AN ACTION GROUP OF MENU ITEMS (A.K.A MEGA-MENUS)?
+
+The client-user should be able to create groups for menu items in the menu. Groups are not interactable.
+
```

Comment:
An important interaction decision to make about groups is whether or not they are part of the "filtering" interaction. One possible nicety is that a user can filter by the group, and then see all of the options under that group alone.